hist/hist/inc/TH3.h

```diff
@@ -150,7 +159,7 @@ class TH3 : public TH1, public TAtt3D {
       return h.DoProject2D(name, title, projX,projY, computeErrors, originalRange, useUF, useOF);
    }
 
-   ClassDef(TH3,5)  //3-Dim histogram base class
+   ClassDef(TH3,6)  //3-Dim histogram base class
```

Comment:
Note: You only need to increase the version number is the list of persistent data member from this class or one of its base classes changes.

Reply:
Right. That was too quick.
I have reverted the patch, making sure that none of the new members is persistent.
Thanks!